packages/components/src/components/dropdown-select/dropdown-select.tsx

```diff
@@ -422,7 +422,7 @@ export class DropdownSelect {
               {...(this.helperText ? ariaDescribedByAttr : {})}
               {...(this.invalid ? { 'aria-invalid': 'true' } : {})}
             >
-              {ValueElement}
+              <span part="combobox_value">{ValueElement}</span>
```

Comment:
I think this should be `combobox-value` with a regular dash (kebab-case) 🧐

Reply:
That definitely makes sense, I will change it. Thank you 🤓